packages/zk-transfer-web/src/components/Token.tsx

```diff
@@ -46,7 +46,7 @@ const TokenInput = (props: TokenPropType) => {
         <span className="px-1 text-base">{props.secondValue}</span>
       ) : (
         <button
-          className="appearance-none"
+          className="appearance-none focus:bg-gray-200 focus:outline-none"
```

Comment:
what are the additional CSS classes needed here? just curious.

Reply:
On Chromium browsers i see an orange outline on the `input` and `button` elements on focus. So i added the css to prevent.

Reply:
great that makes sense and is a good change. The arrow buttons shouldn't be outlined, just the amount inputs.